Data/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ endif(ENABLE_DATA_MYSQL)
 if(ENABLE_DATA_POSTGRESQL)
     find_package(PostgreSQL)
     if(POSTGRESQL_FOUND)
```

Comment:
Sorry, I'm talking about the `PostgreSQL_FOUND`. As you correctly pointed out the variables are case sensitive. And according to the [documentation](https://github.com/Kitware/CMake/blob/master/Modules/FindPostgreSQL.cmake#L46) this should be `PostgreSQL_FOUND` not `POSTGRESQL_FOUND`.